src/main/java/dev/snowdrop/release/App.java

```diff
@@ -259,11 +301,11 @@ public void updateBuildConfig(
         final String gitFullRef = String.format("%s/sb-%s.%s.x", gitRef, releaseMMF[0], releaseMMF[1]);
         Release releaseObj = factory.createFromGitRef(gitFullRef, false, true, release);
 
-        GitConfig config = GitConfig.gitlabConfig("snowdrop/build-configurations", release, gluser, gltoken);
+        GitConfig config = GitConfig.gitlabConfig(release, gluser, gltoken, buildConfigForkRepoName, Optional.of(String.format("sb-%s.%s.x", releaseMMF[0], releaseMMF[1])));
         git.initRepository(config);
 
-        git.commitAndPush("chore: update " + release + " release issues' key [release-manager]", config, repo -> buildConfigUpdateService
-            .updateBuildConfig(repo, releaseObj, release, qualifier, milestone));
+        git.commitAndPush("chore: update " + release + " release issues' key [issues-manager]", config, repo -> Stream.of(buildConfigUpdateService
```

Comment:
Same remark, shouldn't be `release-manager`?

Reply:
changes implemented